src/__tests__/__snapshots__/downshift.aria.js.snap

```diff
@@ -0,0 +1,107 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`basic snapshot 1`] = `
+<div
+  aria-expanded="false"
+  aria-haspopup="listbox"
+  aria-labelledby="downshift-0-label"
+  aria-owns="downshift-0-menu"
+  role="combobox"
+>
+  <label
+    data-testid="label"
+    for="downshift-0-input"
+    id="downshift-0-label"
+  >
+    label
+  </label>
+  <input
+    aria-autocomplete="list"
+    aria-controls="downshift-0-menu"
+    aria-expanded="false"
+    aria-labelledby="downshift-0-label"
+    autocomplete="off"
+    data-testid="input"
+    id="downshift-0-input"
+    role="listbox"
```

Comment:
I'd recommend removing the `role=listbox` attribute here. It's throwing an error in the axe-core Chrome plugin, and is declared further down on your unordered list.

I'm using https://www.w3.org/TR/wai-aria-1.1/#combobox as my guide and cross-reference for any errors thrown by axe-core.

Reply:
You absolutely need to remove this role. At its core, a Combobox is a composite of a `textbox` and a "popup", most commonly a `listbox`. The textbox is important, it's what takes the value once set, what indicates you can type and edit, the thing that takes focus when interacting with the control, etc. You can just remove it as an `ipnut` has an implicit role of `textbox`, or you can explicitly set it to `role="textbox`